guidelines/content-guidelines/09-add-documentation-topic.md

```diff
@@ -3,86 +3,85 @@ title: Add new documentation to the website
 ---
 ​
 This document explains how to add a new topic to the Kyma documentation to render it on the [Kyma website](https://kyma-project.io). It also describes how to modify an existing topic if you want OpenAPI specifications to show in a given topic's documentation.
-​
+
 ## Add a new documentation topic
-​
+
 Follow these steps:
 
 1. Create a pull request with `.md` files for the new documentation topic. Place the `.md` files under a new `docs` subfolder in the `kyma` repository, such as `docs/api-gateway-v2/`.
 
-2. In the same PR, create a `.yaml` file under the [`templates`](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/docs/charts/content-ui/templates) folder to add a [ClusterDocsTopic CR](https://kyma-project.io/docs/components/headless-cms/#custom-resource-cluster-docs-topic) for your topic. For example, if you add a ClusterDocsTopic CR for the API Gateway v2 component, name it `docs-components-api-gateway-v2-cdt.yaml`. ​
+2. In the same PR, create a `.yaml` file under the [`templates`](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/docs/charts/content-ui/templates) folder to add a [ClusterAssetGroup CR](https://kyma-project.io/docs/components/rafter/#custom-resource-cluster-asset-group) for your topic. For example, if you add a ClusterAssetGroup CR for the API Gateway v2 component, name it `docs-components-api-gateway-v2-cag.yaml`. ​
```

Comment:
Path will change:

```md
2. In the same PR, create a `.yaml` file under the [`templates`](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/content-ui/templates) folder to add a [ClusterAssetGroup CR](https://kyma-project.io/docs/components/rafter/#custom-resource-cluster-asset-group) for your topic. For example, if you add a ClusterAssetGroup CR for the API Gateway v2 component, name it `docs-components-api-gateway-v2-cag.yaml`. ​
```

Reply:
Link `https://github.com/kyma-project/kyma/tree/master/resources/core/charts/docs/charts/content-ui/templates` is correct. Sorry.